downstream/modules/builder/ref-build-args-base-image.adoc

```diff
@@ -9,14 +9,13 @@ The `build_arg_defaults` section of the definition file is a dictionary whose ke
 | Value | Description
 
 | `ANSIBLE_GALAXY_CLI_COLLECTION_OPTS`
-| * Allows the user to pass the `–pre` flag to enable the installation of pre-releases collections
-* `-c` is the equivalent of setting `verify_ssl` to `false`
+| Allows the user to pass arbitrary arguments to the ansible-galaxy CLI during the collection installation phase. For example, the –pre flag to enable the installation of pre-release collections, or -c  to disable verification of the server's SSL certificate.
 
 | `EE_BASE_IMAGE`
-| Specifies the parent image for the {ExecEnvNameSing}, enabling a new image to be built that is based off of an already-existing image
+| Specifies the parent image for the automation execution environment, enabling a new image to be built that is based off of an already-existing image. This is typically a supported execution environment base image like ee-minimal or ee-supported, but can also be an execution environment image you've created previously and want to customize further.
```

Comment:
```suggestion
| Specifies the parent image for the automation execution environment, enabling a new image to be built that is based off of an already-existing image. This is typically a supported execution environment base image like ee-minimal or ee-supported, but it can also be an execution environment image that you've created previously and want to customize further.
```

Reply:
Fixed.